### ISSUE TITLE
Update Block Explorer to bts.ai

### ISSUE THREAD

**Issue by bangzi1001** [bitshares/bitshares.org#112]
Problem: https://bitshares-explorer.io is not working

Suggestion: Change Block Explorer link to  https://bts.ai

Block Explorer Link at Footer:
![2020-04-27_18h58_12](https://user-images.githubusercontent.com/36911788/80365309-efb27a00-88b9-11ea-93ed-2efe647578d3.jpg)

https://bitshares-explorer.io show no data:
![image](https://user-images.githubusercontent.com/36911788/80365178-c1cd3580-88b9-11ea-9079-46afa5fbb99e.png)

**Comment by oxarbitrage** [comment]
I agree on the change, the open-explorer needs an elasticsearch server that i no longer have since a while. There were a few other ES servers from infra workers up but those are now gone too.

In a related subject i don't think the explorer itself is very important but i do think bitshares should have elasticsearch and kibana servers available, unfortunately these are expensive and hard to mantain.
